src/index.ts

```diff
@@ -40,8 +40,7 @@ extension.Add("cli.common", async autoRestApi => {
         });
 
         // emit a file (all input files concatenated)
-
-        autoRestApi.WriteFile("myfolder/concatcommon.txt", inputFiles.join("\n---\n"));
+        autoRestApi.WriteFile("code-model-v4-cli.yaml", inputFiles[1]);
```

Comment:
i think you should try to find input file by filename, what if sequence changes?

Reply:
changed